torch/testing/_asserts.py

```diff
@@ -154,58 +158,71 @@ def _equalize_attributes(a: torch.Tensor, b: torch.Tensor) -> Tuple[torch.Tensor
     return a, b
 
 
-_Trace = namedtuple("Trace", ("total", "abs", "rel", "idx", "diff", "a", "b"))
+_Trace = namedtuple(
```

Comment:
This is cool, especially since it will be easy to extend in the future if we like